### PR TITLE
autoinstall: fix issues in autoinstall roundtrip

### PR DESCRIPTION
**Pull request by dbungert**
Fix autoinstall output for kernel & snaplist.
This is necessary but not sufficient to roundtrip an autoinstall file.

### CODE REVIEW

**Comment by mwhudson** [comment]
Argl let's just fix how the snap stuff is generated please?

**Comment by mwhudson** [comment]
Argh sorry to be a pain here but I think we probably need to keep compatibility in the answers file here, there is an automated test case that uses is_classic. Or I guess we can fix this as soon as this branch moves to stable... maybe that's OK.

**Review by mwhudson — APPROVED** [review]

**Comment by mwhudson** [comment]
Pushed an attempt at that, feel free to merge if you like that.

File: subiquity/server/controllers/kernel.py [hunk 1]
```diff
@@ -75,8 +75,4 @@ def load_autoinstall_data(self, data):
         self.model.metapkg_name = package
 
     def make_autoinstall(self):
-        return {
-            'kernel': {
```
Comment (RobQuistNL) [inline subiquity/server/controllers/kernel.py:79]: Thank you :) came here to fix this bugger..

**Pull request merged**